scipy/misc/_ppool.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import division, print_function, absolute_import
+from multiprocessing import Pool, cpu_count
+
+__all__ = ["PPool"]
+
+class PPool(object):
```

Comment:
Why not just extend `multiprocessing.pool.Pool`? In that way we don't need to have the `map` method, it's inherited, as are others.